sycl/plugins/cuda/pi_cuda.hpp

```diff
@@ -382,6 +382,7 @@ struct _pi_queue {
 
   std::vector<native_type> compute_streams_;
   std::vector<native_type> transfer_streams_;
+  std::vector<char> delay_compute_;
```

Comment:
It would make sense to use `std::vector<bool>` instead as it may be specialized for [space efficiency](https://en.cppreference.com/w/cpp/container/vector_bool).